BOJ/240117_회의실_배정/kwaksj329.js

```diff
@@ -0,0 +1,23 @@
+const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
+const N = Number(input[0]);
+const meeting = input.slice(1, N + 1).reduce((acc, curr) => {
+  acc.push(curr.split(' ').map(Number));
+  return acc;
+}, []);
```

Comment:
```suggestion
const meeting = input.slice(1).map((line) => line.split(' ').map(Number));
```

`reduce`를 사용하지 않아도 괜찮을 것 같아요!